database/insertData/05_review_assignments.js

```diff
@@ -40,9 +45,14 @@ exports.queries = [
       input: {
         reviewAssignment: {
           applicationId: 4000
-          assignerId: 8
-          reviewerId: 7
           stageId: 5
+          userToReviewerId: {
+            connectByUsername: {
+              username: "testReviewer2"
+            }
+          }
+          status: AVAILABLE_FOR_SELF_ASSIGNMENT
+          availableSectionsIds: [1006]
```

Comment:
Just checking -- are these available sections consistent with those user's actual permissions?

Reply:
Hmm, no! I was only trying to display that it should be used like that. But I guess we can update the file `02_users` on the inserData folder to add restrictions to the permission_template? I'll work on that tomorrow.

Reply:
It's not that big a deal, it's just that eventually this table will be completely generated from an Action script, in which case all those things will be enforced.

Reply:
Alright, I'll be leaving like this for now.